scripts/pre_commit_hook.py

```diff
@@ -139,13 +136,15 @@ def main():
         sys.exit(0)
 
     python_utils.PRINT('Running pre-commit check for package-lock.json ...')
-    if _does_diff_include_package_lock_file_and_no_package_file():
+    if _does_diff_include_package_lock_file():
         python_utils.PRINT(
-            'This commit only changes package-lock.json without '
-            'any change in package.json. Therefore changes in '
-            'package-lock.json will be automatically reverted.')
-        python_utils.PRINT('Reverting changes in package-lock.json ...')
-        _revert_changes_in_package_lock_file()
+            'Oppia utilizies Yarn to manage npm packages. Please use Yarn '
```

Comment:
utilizies --> utilize

Perhaps give more info on how to "use Yarn" -- i.e. edit which file exactly?

Reply:
Done! As for the "how to use Yarn" part, I've linked a url to that.